docs/troubleshooting.md

```diff
@@ -3,28 +3,29 @@
+| Empty directory doesn't obey `--kernel-list-cache-ttl-secs` ttl.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | In case a new file is added to the empty directory remotely, outside of the mount, the client will not be able to see the new file in listing even if ttl is expired.<br></br>You need to create a new file/directory from the mount to see the newly added remote files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |                                                                                                      
```

Comment:
Shall we add that kernel list cache should only be used for read only workloads?